app/src/main/java/org/oppia/android/app/story/StoryActivity.kt

```diff
@@ -65,7 +64,6 @@ class StoryActivity :
     startActivity(
       ExplorationActivity.createExplorationActivityIntent(
         this,
-        profileId,
```

Comment:
Here and routeToResumeLesson below, at which point is the intent decorated with the profileId as a replacement?